pkg/controller/acicontainersoperator.go

```diff
@@ -138,6 +181,22 @@ func (c *Controller) Run(stopCh <-chan struct{}) {
 
 	c.Logger.Info("Controller.Run: initiating")
 
+	log.Info("Checking if Custom Resource already present")
+	err := c.GetAciContainersOperatorCR()
+	if err != nil {
+		log.Info("Not Present ..Creating Custom Resource")
+		err = c.CreateAciContainersOperatorCR()
+		if err != nil {
+			log.Error("Function returned error",err)
+			return
+		}
+	}
+
+        if err == nil {
+
+                log.Info("Custom Resource already present")
```

Comment:
Is it possible that c.GetAciContainersOperatorCR() fails for some other reason?

Reply:
This is the passing condition , error handled in earlier leg